wdl/ApplyManualVariantFilter.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+import "Structs.wdl"
+
+workflow ApplyManualVariantFilter {
+  input {
+    String prefix
+    File vcf
+    File? vcf_index
+    String filter_name
+    String bcftools_filter  # supplied to bcftools view -e "<filter>"
+
+    String sv_base_mini_docker
+    RuntimeAttr? runtime_attr_hard_filter_vcf
+  }
+
+  call HardFilterVcf {
+    input:
+      prefix = prefix,
+      vcf = vcf,
+      vcf_index = vcf_index,
+      filter_name = filter_name,
+      bcftools_filter = bcftools_filter,
+      sv_base_mini_docker = sv_base_mini_docker,
+      runtime_attr_override = runtime_attr_hard_filter_vcf
+  }
+
+  output {
+    File manual_filtered_vcf = HardFilterVcf.hard_filtered_vcf
+    File manual_filtered_vcf_index = HardFilterVcf.hard_filtered_vcf_index
+  }
+}
+
+
+task HardFilterVcf {
+  input {
+    String prefix
+    File vcf
+    File? vcf_index
+    String filter_name
+    String bcftools_filter
+
+    String sv_base_mini_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  String hard_filtered_vcf_name = "~{prefix}.~{filter_name}.vcf.gz"
+
+  # Disk must be scaled proportionally to the size of the VCF
+  Float input_size = size(vcf, "GiB")
+  RuntimeAttr default_attr = object {
+    mem_gb: 3.75,
+    disk_gb: ceil(10.0 + (input_size * 2)),
+    cpu_cores: 1,
+    preemptible_tries: 3,
+    max_retries: 1,
+    boot_disk_gb: 10
+  }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  command <<<
+
+    set -euo pipefail
+
+    bcftools view -e "~{bcftools_filter}" ~{vcf} -Oz -o "~{hard_filtered_vcf_name}"
```

Comment:
```suggestion
    bcftools view -e '~{bcftools_filter}' ~{vcf} -Oz -o "~{hard_filtered_vcf_name}"
```
Should use single quotes so that special characters such as `@` and `*` won't be interpreted by the shell. I believe then you also have to modify the filter expression in the input json to use double quotes for strings, escaped with a backslash i.e. `"SVTYPE==\"DEL\" …"`.